integration/cursor/getmore_test.go

```diff
@@ -467,6 +467,11 @@ func TestGetMoreCommandBatchSize(t *testing.T) {
 				require.Equal(t, i-1, cursor.RemainingBatchLength())
 			}
 
+			if setup.IsYugabyteDB(t) {
+				// TODO https://github.com/yugabyte/yugabyte-db/issues/27989
+				t.Skip("https://github.com/yugabyte/yugabyte-db/issues/27989")
+			}
```

Comment:
Could we use expected failures?

Reply:
Unfortunately no. Because it doesn't fail, YugabyteDB returns the same batch repeatedly again and again, which causes the test to never exit, until the timeout.